### PR TITLE
blog: announce async-backtrace

### PR DESCRIPTION
**Pull request by jswrenn**
Announcement post for initial release of async-backtrace crate.

@carllerche

### CODE REVIEW

**Comment by carllerche** [comment]
Rendered: https://deploy-preview-676--tokio.netlify.app/blog/2022-10-announcing-async-backtrace

**Comment by carllerche** [comment]
I think we should expand on how this crate relates to other projects. Readers will be asking themselves two questions: "How does this compare to X (`tracing`, "Tokio Console", ...)  other lib" and "How does this fit into my workflow".

The answer to the first question is that it is complementary and is a "building block" utility (better wording needed). Eventually, Tokio Console could use this.

**Comment by LucioFranco** [comment]
The example feels kinda hard to follow maybe include more of the example and some more complex real world examples? Like can you show this with hyper?

**Comment by jswrenn** [comment]
I've updated the post to include a totally self-contained example, and updated the repo to contain many more examples.

Since the usage guidelines of the crate are fairly simple — apply `#[async_backtrace::framed]` as liberally as possible —  I don't think the cognitive load of introducing additional libraries would be justified by its demonstrative benefits. The usage would be the same, but aspects of the example would be unfamiliar to anyone without experience with hyper.

**Review by carllerche — APPROVED** [review]
